lib/core/src/rodsPath.cpp

```diff
@@ -555,6 +555,39 @@ clearRodsPath( rodsPath_t *rodsPath ) {
     return;
 }
 
+auto clearRodsPathInp( rodsPathInp_t* path ) -> void {
+  if (nullptr == path) {
+    return;
+  }
+
+  for (int i{}; i < path->numSrc; i++) {
+    clearRodsPath(&path->srcPath[i]);
+    clearRodsPath(&path->targPath[i]);
+  }
+  clearRodsPath(path->destPath);
+
+  memset(path, 0, sizeof(rodsPathInp_t));
+}
```

Comment:
I think this function is incorrect. We lose the data pointed by `srcPath`, `targPath`, and `destPath`.

The only place this function is used should call `freeRodsPathInpMembers` instead.

Reply:
This might also explain why there are smaller leaks still. Will replace and check.

Reply:
This does not fix the persistent few leaks, but it does allow us to only have one new function (rather than two).

Will not be including the `clearRodsPathInp(...)` function.